prac_07/project_managment.py

```diff
@@ -0,0 +1,63 @@
+"""
+CP1404 | project and project management | Liam Eime
+Estimate time = 1.5hrs
+Actual time =
+"""
+
+import datetime
```

Comment:
If you use `from datetime import datetime`, you only can use `datetime.strptime()` instead of `datetime.datetime.strptime()`. Otherwise not much of a difference